src/modules/Lan.rb

```diff
@@ -542,6 +542,7 @@ def Write
       # Progress step 3 - multiple devices may be present, really plural
       ProgressNextStage(_("Writing device configuration..."))
       LanItems.write
+      return false if PackagesInstall(Packages()) != :next
```

Comment:
Is it really needed?, Isn't it handled by the sequence itsef?

Reply:
just wanted to stop Lan::Write earlier ... otherwise it will try to write Routing, DNS, ...

Reply:
Then seems it should be like an extra step about checking missed dependencies.

Reply:
fixed ... I did a mistake in my tests before. thaks